app/services/weekly_stats_summary.rb

```diff
@@ -38,6 +39,10 @@ def offers_made(period, year)
     Offer.joins(:application_choice).where('application_choice.current_recruitment_cycle_year': year, created_at: period).count
   end
 
+  def offers_accepted(period, year)
```

Comment:
is `period` a sufficiently meaningful parameter name here?

Reply:
reading the code properly, i think it's ok. maybe `offers_accepted_in` would be a better method name, and `recruitment_cycle` a better name for the other param.